plugins/modules/rocketchat.py

```diff
@@ -100,6 +100,14 @@
     elements: dict
     description:
       - Define a list of attachments.
+  is_pre740:
+    description:
+      - If V(true), the payload matches Rocket Chat prior to 7.4.0 format.
+      - The default value of the parameter can be set to false in a few months' time.
+      - This parameter will be removed in a future release when Rocket Chat 7.4.0 becomes the minimum supported version.
```

Comment:
I think the official spelling is Rocket.Chat with a period:
```suggestion
      - This parameter will be removed in a future release when Rocket.Chat 7.4.0 becomes the minimum supported version.
```
(Also it might be that this has been already working with Rocket.Chat for a long time, and it was just that they dropped support for the old way in 7.4.0 - maybe because they thought that nobody is using it anymore since forever.)